From 9432735689a47f97da98924b0705816a5e147f4e Mon Sep 17 00:00:00 2001
From: Yclept Nemo <orbisvicis@gmail.com>
Date: Thu, 23 Jan 2014 10:39:16 -0500
Subject: [PATCH] CMake: change variables shown in configuration GUI

    hide: CMAKE_INSTALL_PREFIX
    show: BUILD_SHARED_LIBS (default TRUE), CMAKE_SKIP_BUILD_RPATH
          (default FALSE)
---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0e55a45..4637deb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,11 @@ project(flandmark)
 set(flandmark_VERSION_MAJOR 1)
 set(flandmark_VERSION_MINOR 7)
 
+# configure which options are shown in the gui
+MARK_AS_ADVANCED(CMAKE_INSTALL_PREFIX)
+SET(BUILD_SHARED_LIBS "TRUE" CACHE BOOL "Global flag to cause add_library to create shared libraries if on.")
+SET(CMAKE_SKIP_BUILD_RPATH "FALSE" CACHE BOOL "Do not include RPATHs in the build tree.")
+
 if(NOT CMAKE_BUILD_TYPE)
 	set(CMAKE_BUILD_TYPE RELEASE CACHE STRING
 		"Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel."
-- 
1.8.2.2

